source/regex.tex

```diff
@@ -3692,19 +3692,14 @@
 \tcode{upper}, \tcode{xdigit}, \tcode{d}, \tcode{s}, \tcode{w}.
 In addition the following expressions shall be equivalent:
 
-\begin{codeblock}
-\d and [[:digit:]]
-
-\D and [^[:digit:]]
-
-\s and [[:space:]]
-
-\S and [^[:space:]]
-
-\w and [_[:alnum:]]
-
-\W and [^_[:alnum:]]
-\end{codeblock}
+\begin{tabular}{@{\quad}l@{and}l}
+  \tcode{\textbackslash d } & \tcode{ [[:digit:]]} \\[3mm]
```

Comment:
The "3mm" extra spacing seems oddly arbitrary.

Would a smaller change be to escape from the "codeblock" environment and just write @and@ to get a prose-font "and"?

Reply:
I'd like to see that option explored, too. Generally, I tried to get the vertical whitespace be the same everwhere as much as possible, so I'd be reluctant to introduce an explicit special size here.

Reply:
Ah, after trying a bunch of different ways to escape the codeblock environment, I gave up and used a tabular instead, but @ seems to work like a charm, thanks! :) Updating patch.